class-tgm-plugin-activation.php

```diff
@@ -2093,6 +2095,19 @@ public static function get_instance() {
 
 			return self::$instance;
 		}
+
+		/**
+		 * Adds CSS to admin head.
+		 *
+		 * @since 2.6.2
+		 */
+		function admin_css() {
+			echo '<style>
```

Comment:
I'd suggest starting with a new line, so this code will present aligned in the HTML,

```php
	echo '
	<style>
```

And maybe indent the css nicely as well ?